OxfordBodleian/Juel-Jensen/BDLaethd23.xml

```diff
@@ -154,21 +260,201 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <add place="above">የ</add>ማርያም፡ ገበዝ፡ ያውቃል፡ ይጠይቁ፡ ግብር፡ 
+                                        የለበትም፡ እ<hi rend="ligature">ግዜ</hi>ር፡ ያስየዎ፡ አምላክ፡ ያመልክተዎ፡ 
+                                        የኔ፡ ጌታ፡ ከቤቱ፡ 
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a5">
+                                    <desc type="RecordTransaction">Note in black ink recording the purchase of the land Kok by the
+                                        <foreign xml:lang="am">fitāwrāri</foreign> <persName ref="PRS14793Mahsantu"/> 
+                                        for 30 <foreign xml:lang="am">waqet</foreign>.
+                                        The creditor (<foreign xml:lang="am">bālaʿədā</foreign>) was <persName ref="PRS14792WaldaLeul"/>, 
+                                        the seller (<foreign xml:lang="am">šayāč̣</foreign>) was <persName ref="PRS14795Meherka"/>,
+                                        the guarantor (<foreign xml:lang="am">madən</foreign>) was <persName ref="PRS14796Wasan"/>,
+                                        and the purchase took place
+                                        during the period of office (<foreign xml:lang="am">gabazənnat</foreign>) 
+                                        of the priest <persName ref="PRS14797Ramha"/>.
+                                    </desc>
+                                    <locus target="#49va"/>
+                                    <q xml:lang="am">
+                                        ፊታውራሪ፡ ማኅፀንቱ፡ የገዙ<del rend="effaced">ት</del>፡ 
+                                        ምድር፡ ኮክ፡ ዋጋው፡ ፴ ወቄት፡ ባለዕዳው፡ ወልደ፡ ልዑል፡ ኂሩተ፡ አብ፡ 
+                                        ሺያጩ፡ ወልደ፡ ልዑል፡ ምሕርካ፡ መድኑ፡ ወሰን፡ ካህኑ፡ በቄስ፡ ረምሐ፡ ገበዝነት፡
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a6">
+                                    <desc type="GuestText">Note in blue pen containing four very succinct miracles of 
+                                        <foreign xml:lang="gez">ʾAbbā</foreign> ʾAragāwi.</desc>
+                                    <locus from="70ra" to="70rb"/>
+                                    <q xml:lang="am">
+                                    ስለአቡነ፡ አረጋዊ፡ የተደረገው፡ ተአምር፡ አንደኛ፡ በአጼ፡ ዮሐንስ፡ ይህ፡ ገድል፡ እና፡ 
+                                    ይህ፡ መስቀል<supplied reason="omitted">፡</supplied> ትግሬው፡ ወስዶት፡ 
+                                    ተመለሰ፡ ፪ተኛ፡ ይኽው፡ ገድል፡ ዲያቆን፡ ወስዶ፡ ሸጦት፡ ያው፡ ዲያቆን፡ ሲቀድስ፡ 
+                                    ጋአን፡ ጥሎት፡ በዚያው፡ ሞተ፡ ፫ተኛ፡ በጣልያን፡ ጊዜ፡ አማሲን፡ ይህኑ፡ ገድል፡ 
+                                    ወስዶ፡ አማስኑ፡ ይክ<supplied reason="omitted">፡</supplied> ገድል፡ እንደስት፡ 
+                                    ፈጀን፡ ሲል፡ ፫ አጠገራ፡ ብር፡ ጭም?፡ መልስታል፡ ፬ኛ፡ ጥር፡ ፳?፡ ቀን፡ ፶፯ ዓመ፡ ም፡ 
```

Comment:
፳?፡
Is there a non legible character after ፳?

Reply:
Yes, thanks for noticing this!